ledger/sandbox/src/main/resources/db/migration/postgres/V21__Stable_offsets.sql

```diff
@@ -0,0 +1,31 @@
+-- Copyright (c) 2019 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+---------------------------------------------------------------------------------------------------
+-- V20: Stable Offsets
```

Comment:
```suggestion
-- V21: Stable Offsets
```

Reply:
It's probably not necessary to repeat the information in the file name here.